.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,106 @@
+# To test: gh workflow run 'Build Docker Image' --ref kwannoel/workflow-update-branch
+name: Build Docker Image
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag for the Docker image: vX.Y.Z-<tag>-<commit_sha>'
```

Comment:
I think the term "tag" is already widely recognized in the scope of Docker, to represent the full `vX.Y.Z-foo-1234567`, rather than just `foo`. Can we change to another term to avoid confusion?